go/config/defaults/testnet-launcher/2-validator.yaml

```diff
@@ -0,0 +1,20 @@
+node:
+   name: validator
+   nodeType: validator
+   isGenesis: false
+   id: 0x2f7fCaA34b38871560DaAD6Db4596860744e1e8A
+   privateKey: ebca545772d6438bbbe1a16afbed455733eccf96157b52384f1722ea65ccfa89
```

Comment:
Are we going to leave this as is? Perhaps this field should either be a key or somewhere to load it from ( or off chain signing )
Maybe a todo with a ticket